.build/run-python-dtests.sh

```diff
@@ -164,10 +165,13 @@ fi
 
 # merge all unit xml files into one, and print summary test numbers
 pushd ${CASSANDRA_DIR}/ >/dev/null
-# remove <testsuites> wrapping elements. `ant generate-unified-test-report` doesn't like it`
+# remove <testsuites> wrapping elements. ant generate-test-report` doesn't like it, and update testsuite name
 sed -r "s/<[\/]?testsuites>//g" ${DIST_DIR}/test/output/nosetests.xml > ${TMPDIR}/nosetests.xml
 cat ${TMPDIR}/nosetests.xml > ${DIST_DIR}/test/output/nosetests.xml
-ant -quiet -silent generate-unified-test-report
+sed "s/testsuite name=\"Cassandra dtests\"/testsuite name=\"${DTEST_TARGET}_jdk${java_version}_python${python_version}_cython${cython}_$(uname -m)${SPLIT_STRING}\"/g" ${DIST_DIR}/test/output/nosetests.xml > ${TMPDIR}/nosetests.xml
+cat ${TMPDIR}/nosetests.xml > ${DIST_DIR}/test/output/nosetests.xml
```

Comment:
Let's declare `${DIST_DIR}/test/output/` as a constant, it's used many times throughout the file

Reply:
I think we can also consider moving it to the `build.xml`

Reply:
not sure exactly what you mean by moving it to `build.xml` (but i'd suggest doing whatever that is as a separate ticket – this patch is blocking the 5.0-beta2 release)

Reply:
pushed, check new commits.